### ISSUE TITLE
Don't ask to upload?

### ISSUE THREAD

**Issue by jdumas** [carnager/teiler#40]
Hi,

Nice little program, I like how easy it is to use. I was wondering if it was possible to disable the question "Upload? Yes/No" after taking a screencast. I couldn't care less about uploading my videos to whatever website, and I've already set the option `always_ul=0` in the setting file, but I haven't seen a way to *not* ask the question at all (and I'm afraid to break something if I try to hack the script directly).

Can this be done or added as an option?

**Comment by carnager** [comment]
Yeah, not at the moment. But I am not happy with some of the workflow anyway, so chances are this will change. Ideally it would never ask any questions, but the only way to do this with any possible use case is bloated menus or multiple hotkeys. Both are not ideal too. But yea, it's on my radar.

**Comment by jdumas** [comment]
Ok. Please let me know when you get to it =)
I think an option in the config file to skip this question would be fine imho (at least for the usage that I have of it).

**Comment by dkasak** [comment]
I'd also love this. teiler popping up to ask about upload each time is annoying.

**Comment by cjbassi** [comment]
Also, as far as workflow goes, I think it would be better to change the screenshots menu so that choosing the destination is done through another rofi menu rather than pressing Alt+1 or Alt+2.
